supplementary/devices/index.rst

```diff
@@ -26,12 +26,13 @@ All of these should only be document in the exact same way here and **only an ov
    T00F
    titan
    vegeta
+   wt88047
```

Comment:
Either remove this from the index or add the supplementary page, it's causing a warning (and your build fails)